hooks/001-extra-packages.chroot

```diff
@@ -277,7 +277,7 @@ case "$(dpkg --print-architecture)" in
         # Copy nvidia config files and dependencies (sources for this come from
         # Canonical, not from Nvidia, see
         # https://github.com/canonical/nvidia-graphics-drivers).
-        nvidia_common_pkg=nvidia-kernel-common-570
+        nvidia_common_pkg=nvidia-kernel-common-580
```

Comment:
does this need a matching initramfs update?

Reply:
I do not think we have those files in core-initrd.